### PR TITLE
Typescript-eslint issues and storybook improvements

### PR DESCRIPTION
**Pull request by irvelervel**
**Changes proposed in this pull request:**
- Fixes typescript-eslint issues
- Improves toaster storybook
- Adds confirm storybook

### CODE REVIEW

**Comment by fox1t** [comment]
LGTM

File: package.json [hunk 1]
```diff
@@ -16,6 +16,8 @@
     "dev": "tsdx watch",
     "prebuild": "npm run test",
     "build": "tsdx build",
+    "prepublish": "npm run build",
```
Comment (jackcmeyer) [inline package.json]: I think that this line needs to go away

Comment (fox1t) [reply package.json]: Why?

Comment (fox1t) [reply package.json]: Maybe prepublishOnly is better?

Comment (jackcmeyer) [reply package.json]: we removed this in hospitalrun/components#48

Comment (tehKapa) [reply package.json]: My fault!

Comment (jackcmeyer) [reply package.json]: no worries :)

**Pull request merged**